roles/ceph-nfs/templates/ceph-nfs.service.j2

```diff
@@ -11,7 +11,8 @@ ExecStart=/usr/bin/docker run --rm --net=host \
   {% if not containerized_deployment_with_kv -%}
   -v /var/lib/ceph:/var/lib/ceph \
   -v /etc/ceph:/etc/ceph \
-  -v /var/lib/ganesha:/var/lib/ganesha \
+  -v /var/lib/nfs/ganesha:/var/lib/nfs/ganesha \
+  -v /var/log/ganesha:/var/log/ganesha \
```

Comment:
👎  logs from the container are on stdout already, so they are on journald and appear on `docker logs` too.